subprojects/testing-jvm/src/main/java/org/gradle/api/tasks/testing/Test.java

```diff
@@ -1127,13 +1130,33 @@ public long getForkEvery() {
      * By default, Gradle automatically uses a separate JVM when executing tests, so changing this property is usually not necessary.
      * </p>
      *
-     * @param forkEvery The maximum number of test classes. Use null or 0 to specify no maximum.
+     * @param forkEvery The maximum number of test classes. Use 0 to specify no maximum.
+     * @since 8.1
      */
-    public void setForkEvery(@Nullable Long forkEvery) {
-        if (forkEvery != null && forkEvery < 0) {
+    public void setForkEvery(long forkEvery) {
```

Comment:
Should we do this for these other classes?
```
subprojects/core-api/src/main/java/org/gradle/api/file/CopyProcessingSpec.java:    CopyProcessingSpec setFileMode(@Nullable Integer mode);
subprojects/core-api/src/main/java/org/gradle/api/file/CopyProcessingSpec.java:    CopyProcessingSpec setDirMode(@Nullable Integer mode);
subprojects/core/src/main/java/org/gradle/api/tasks/AbstractCopyTask.java:    public AbstractCopyTask setDirMode(@Nullable Integer mode) {
subprojects/core/src/main/java/org/gradle/api/tasks/AbstractCopyTask.java:    public AbstractCopyTask setFileMode(@Nullable Integer mode) {
```

Reply:
No need. Those are fine. The nullability of getters and setters is already symmetrical so the property assignment syntax already works.

Reply:
Ah, OK. Thanks!